docs/protocol/extension_model_repository.md

```diff
@@ -116,8 +116,30 @@ $repository_index_error_response =
 
 The load API requests that a model be loaded into Triton, or reloaded
 if the model is already loaded. A load request is made with an HTTP
-POST to a load endpoint. The HTTP body must be empty. A successful
-load request is indicated by a 200 HTTP status.
+POST to a load endpoint.  The HTTP body may be empty or may contain
+the load request object, identified as $repository_load_request.
+A successful load request is indicated by a 200 HTTP status.
+
+
+```
+$repository_load_request =
+{
+  "parameters" : $parameters #optional
+}
+```
+
+- "parameters" : An object containing zero or more parameters for this
+  request expressed as key/value pairs. See
+  [Parameters](https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/required_api.md#parameters)
+  for more information.
+
+The load API accepts the following parameters:
+
+- "config" : string parameter that contains a JSON representation of the model
```

Comment:
IF a load operation is called on a model with a specified config, what will be the expected behavior?

Reply:
"This config will be used for loading the model instead of the one in
the model directory."

Reply:
@GuanLuo If the load request with a config parameter is issued over a model that is already loaded, my guess is the model with previous config would be unloaded and the re-loaded model will use the config from the request. Am I right? 

May be just nitpicking. If this happens on an ensemble model where the config parameter modifies the composing model. The load API can trigger multiple model unloads as well. May be it is a good idea to call out these cases explicitly in the documentation.